Formula/makensis.rb

```diff
@@ -42,7 +45,11 @@ def install
     end
 
     # Don't strip, see https://github.com/Homebrew/homebrew/issues/28718
-    scons "STRIP=0", "ZLIB_W32=#{@zlib_path}", "SKIPUTILS=NSIS Menu", "makensis"
+    if build.with? "large-strings"
+      scons "NSIS_MAX_STRLEN=8192", "STRIP=0", "ZLIB_W32=#{@zlib_path}", "SKIPUTILS=NSIS Menu", "makensis"
```

Comment:
This is quite repetitive. It would be good to streamline this with an `args` array.